vite.config.js

```diff
@@ -0,0 +1,88 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+import fs from 'fs/promises';
+import path from 'node:path';
+import { fileURLToPath } from 'url';
+import { globSync } from 'glob';
+import pkg from './package.json';
+
+/**
+* Vite configuration
+*/
+export default defineConfig({
+  ...(
+    process.env.NETLIFY ? {
+      build: {
+        rollupOptions: {
+          external: ['__tests__/*', '__mocks__/*'],
+          input: Object.fromEntries(
+            globSync('./demo/src/*.html').map((file) => [
+              // This remove `src/` as well as the file extension from each
+              // file, so e.g. src/nested/foo.js becomes nested/foo
+              path.relative(
+                'demo/src/',
+                file.slice(0, file.length - path.extname(file).length),
+              ),
+              // This expands the relative paths to absolute paths, so e.g.
+              // src/nested/foo becomes /project/src/nested/foo.js
+              fileURLToPath(new URL(file, import.meta.url)),
+            ]),
+          ),
+        },
+        sourcemap: true,
+      },
+    } : {
+      build: {
+        lib: {
+          entry: './src/index.js',
+          fileName: (format) => (format === 'umd' ? 'mirador-dl-plugin.js' : 'mirador-dl-plugin.es.js'),
+          formats: ['es', 'umd'],
+          name: 'MiradorDlPlugin',
+        },
+        rollupOptions: {
+          external: [...Object.keys(pkg.peerDependencies || {}), '__tests__/*', '__mocks__/*'],
+          output: {
+            assetFileNames: 'mirador-dl-plugin.[ext]',
+          },
+        },
+        sourcemap: true,
+      },
+    }
```

Comment:
It looks like there's some copy/pasta from the download plugin here?

Reply:
@cbeer thanks for catching this; fixed.